manifest.json

```diff
@@ -1,54 +1,27 @@
 {
-    "name": "__MSG_extName__",
```

Comment:
Changes to manifest.json are too extensive - I can't look at diff file to see what exactly changed. Please revert formatting (whitespace) changes and keep only stuff relevant to adding Edge functionality. There should be just a few lines changed in this file, not 99% of it.